### ISSUE TITLE
First run generated some weird stats?

### ISSUE THREAD

**Issue by joshghent** [niklas-heer/pocketcasts-stats#104]
Hi there
Discovered this via the Automators forum - absolutely great app! Thanks for making it 👍 
I just did my first run and got the following stats. Do they seem a bit weird or what you are expecting for a first run? The stats in the screenshot seem to be a bit more reasonable.

![Screenshot 2019-10-11 at 16 20 21](https://user-images.githubusercontent.com/2934976/66663768-30f77b00-ec43-11e9-9351-20b3b252387a.png)

**Comment by niklas-heer** [comment]
Hi @joshghent nice to hear that you like it. 
Yes, it looks normal, the first one is basically telling you the delta since you have been using PocketCasts.

If you find it irritating you can just do a second run and then delete the first entry. 😉

**Comment by joshghent** [comment]
Ah I'm with you, thanks a lot! Great tool, really love it! 👍 Be interested if you have any similar automations (same thing with Audible and Spotify perhaps?)

**Comment by niklas-heer** [comment]
@joshghent glad I could help you 🙂 

> Be interested if you have any similar automations (same thing with Audible and Spotify perhaps?)

Funny that you ask. 
Actually I tried to query Audible, but unfortunately, it doesn't have a public API and I also couldn't get authentication to work. (You need to fill in a Captcha, which make automation almost impossible)
Spotify has a public API, but from the brief look I had, it doesn't have a statistics endpoint which you can easily query. So also kind of tricky.

Long story short, but I don't have something which works right now. Maybe in the future, but I will probably post it into the forum 😃

**Comment by joshghent** [comment]
@niklas-heer Sorry to comment on a closed issue.
Regarding the spotify stats, perhaps you can query Last.FM instead? Since you can hook Spotify and LastFM together it sort of acts as a proxy API. 

Suck's about Audible but hardly surprising...

**Comment by niklas-heer** [comment]
@joshghent thank you for the tip with last.fm let’s see if I can develop something. If I do what would be the one thing you would find most usable regarding Spotify statistics? Just play time?

**Comment by joshghent** [comment]
@niklas-heer Yeah I reckon playtime is the key metric as you can get most other stats from LastFM already 👍

**Comment by joshghent** [comment]
@niklas-heer Awesome stuff! Look forward to seeing it! Let me know if you want some help 👍
